docs/spinitron.json

```diff
@@ -753,6 +756,160 @@
           }
         }
       }
+    },
+    "/non-music-items": {
+      "get": {
+        "summary": "Returns non-music items optionally filtered by {start} and/or {end} datetimes",
+        "description": "Get non-music items optionally filtered by a datetime range. Only past non-music items will be returned.\n",
+        "tags": [
+          "Non-music item"
+        ],
+        "parameters": [
+          {
+            "name": "start",
+            "description": "The datetime starting from items must be returned.\n",
+            "in": "query",
+            "schema": {
+              "type": "string",
+              "format": "date-time"
+            }
+          },
+          {
+            "name": "end",
+            "description": "The ending datetime.\n",
+            "in": "query",
+            "schema": {
+              "type": "string",
+              "format": "date-time"
+            }
+          },
+          {
+            "name": "playlist_id",
+            "description": "Filter by playlist",
+            "in": "query",
+            "schema": {
+              "type": "integer"
+            }
+          },
+          {
+            "name": "show_id",
+            "description": "Filter by show",
+            "in": "query",
+            "schema": {
+              "type": "integer"
+            }
+          },
+          {
+            "$ref": "#/components/parameters/limit"
+          },
+          {
+            "$ref": "#/components/parameters/page"
+          },
+          {
+            "$ref": "#/components/parameters/fields"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The Non-music items",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "allOf": [
+                    {
+                      "$ref": "#/components/schemas/BaseIndexResponse"
+                    },
+                    {
+                      "type": "object",
+                      "properties": {
+                        "items": {
+                          "type": "array",
+                          "items": {
+                            "$ref": "#/components/schemas/NonMusicItem"
+                          }
+                        }
+                      }
+                    }
+                  ]
+                }
+              },
+              "application/xml": {
```

Comment:
xml? there's a lot of errors in our docs, isn't there

Reply:
Yes, I saw those XML responses were added everywhere and thought maybe it is not an error, and there is some sense. Should I delete those responses from all requests?

Reply:
probably best to leave it. this repo is just for documentation.